lib/public/index.html

```diff
@@ -11,10 +11,64 @@
     <link rel="shortcut icon" href="/favicon.ico" />
     <link rel="stylesheet" href="/css/src/bootstrap.css">
     <link rel="stylesheet" href="/app.css">
-
+    
     <script src="/js/src/compatibility.js"></script>
 </head>
 
+  <!-- ----------------------------------------------------------------------------------- -->
+  <!-- CodeMirror -->
+
+  <link rel="stylesheet" href="./assets/SmartEditor/codemirror/lib/codemirror.css">
```

Comment:
Do not use relative paths here

Reply:
Any specific reason for that?

Reply:
As using relative paths can break while absolute paths wont.

Reply:
Done as well